scripts.py

```diff
@@ -539,6 +539,8 @@ def refresh_metadata(self, identifier):
             provider = OverdriveBibliographicCoverageProvider
         elif identifier.type==Identifier.AXIS_360_ID:
             provider = Axis360BibliographicCoverageProvider
+        elif identifier.type==Identifier.ENKI_ID:
+            provider = EnkiBibliographicCoverageProvider
```

Comment:
You're right, the Axis import should be required here. This whole conversation sort of shows how this code isn't working and doesn't make sense anymore.